subject_templates_test.go

```
@@ -0,0 +1,90 @@
+
+	"github.com/compliance-framework/agent/runner/proto"
+)
+
+func TestBuildSubjectTemplates(t *testing.T) {
+	templates := buildSubjectTemplates()
+	if len(templates) != 1 {
+		t.Fatalf("expected 1 subject template, got %d", len(templates))
+	}
+
+	tpl := templates[0]
+	if tpl.Name != "aws-ec2-instance" {
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field tpl.Name, use tpl.GetName() instead

(protogetter)